extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -301,4 +302,65 @@ export class RemoteQueriesManager extends DisposableObject {
       queryItem.status = QueryStatus.Failed;
     }
   }
+
+  public async exportVariantAnalysisResults(): Promise<void> {
+    const queryHistoryItem = this.qhm.getCurrentQueryHistoryItem();
+
+    if (!queryHistoryItem || queryHistoryItem.t !== 'remote') {
+      throw new Error('No variant analysis results currently open. To open results, click an item in the query history view.');
+    } else if (!queryHistoryItem.completed) {
+      throw new Error('Variant analysis results are not yet available. Wait for results to be downloaded.');
```

Comment:
I'd skip the last sentence (it's not quite true - the problem is that the actions workflow hasn't finished yet).

```suggestion
      throw new Error('Variant analysis results are not yet available.');
```